docs/sources/write/links/_index.md

```diff
@@ -36,20 +37,24 @@ Use the `docs/reference` shortcode.
 The source is reused as described in [Reuse directories of content with Hugo mounts](https://grafana.com/docs/writers-toolkit/write/reuse-content/reuse-directories/).
 For more information and examples, refer to [`docs/reference` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsreference).
 
-## Link to grafana.com pages
+## Link to `grafana.com` pages
 
 Use a full URL.
 
 {{< admonition type="note" >}}
-The `doc-validator` linter does not check links that use full URLs.
+The `doc-validator` linter doesn't check links that use full URLs.
 {{< /admonition >}}
 
-If you are linking to versioned documentation, use a full URL with version substitution syntax.
-Version substitution is necessary for full URLs to link to the correct version of documentation.
-Usually, this is the current version of documentation.
+If you are linking to versioned documentation, use a full URL with version substitution syntax instead of the version path element.
+For example, in Grafana, use `<GRAFANA_VERSION>` instead of `latest` in the URL `https://grafana.com/docs/grafana/latest/`.
```

Comment:
This kind of caught me out the first time I read it.. .I read it as just use `<GRAFANA_VERSION>` as the URL (despite knowing the correct way).  I don't have a suggestion here... just noting that it's one of those "read carefully" sentences.

Reply:
Mm, thanks for raising this. I think it is more clear in https://grafana.com/docs/writers-toolkit/write/shortcodes/#about-version-substitution:~:text=As%20a%20convention%2C%20use%20the%20name%20of%20the%20target%20project%20all%20upper%2Dcase.%20For%20example%2C%20grafana%20becomes%20GRAFANA%2C%20grafana%2Dcloud%20becomes%20GRAFANA%20CLOUD.

Perhaps that should be here and the "About ..." should more on the behavior under the hood.